### PR TITLE
Update cypress test. Fail login.

### PR DESCRIPTION
**Pull request by DmitriyOparin**
Update cypress test. Fail login.
Add cssSelector for elements.
Fix test if running in Firefox.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/integration/actions_users/issue_1810_login_logout.js [hunk 1]
```diff
@@ -64,9 +78,40 @@ context('When clicking on the Logout button, get the user session closed.', () =
...
+            login(Cypress.env('user'), 'randomPassword123');
+            cy.url().should('include', '/auth/login');
+            cy.get('.cvat-notification-notice-login-failed').should('exist');
+            cy.closeNotification('.cvat-notification-notice-login-failed');
+        });
+
+        it('Incorrect user and incorrect password', () => {
+            login('randomUser123', 'randomPassword123');
+            cy.url().should('include', '/auth/login');
+            cy.get('.cvat-notification-notice-login-failed').should('exist');
```
Comment (bsekachev) [inline tests/cypress/integration/actions_users/issue_1810_login_logout.js:114]: Probably ``cy.closeNotification('.cvat-notification-notice-login-failed');`` is missed?

Comment (DmitriyOparin) [reply tests/cypress/integration/actions_users/issue_1810_login_logout.js:102]: Optional to close notice window in last step, but it be present.

File: tests/cypress/integration/actions_users/issue_1810_login_logout.js [hunk 2]
```diff
@@ -71,5 +75,32 @@ context('When clicking on the Logout button, get the user session closed.', () =
...
+            cy.logout();
+            if (Cypress.browser.family !== 'chromium') {
+                cy.get('.cvat-modal-unsupported-platform-warning').within(() => {
+                    cy.contains('button', 'OK').click();
+                });
+            }
+            login('randomUser123', Cypress.env('password'));
+            cy.url().should('include', '/auth/login');
+            cy.get('.cvat-notification-notice-login-failed').should('exist');
+            cy.closeNotification('.cvat-notification-notice-login-failed');
```
Comment (bsekachev) [inline tests/cypress/integration/actions_users/issue_1810_login_logout.js:89]: closeNotification already contains cy.get(className). Do we need this duplication? (the same in the code below)

Comment (DmitriyOparin) [reply tests/cypress/integration/actions_users/issue_1810_login_logout.js]: Unnecessary actions removed.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**